app/main.py

```diff
@@ -1,25 +1,68 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.max_amount = max_amount
+        self.min_amount = min_amount
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: Any, owner: Any) -> Any:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: Any, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError("Value should be integer")
+        if value < self.min_amount or value > self.max_amount:
+            raise ValueError(f"Value should not be less than "
+                             f"{self.min_amount} and greater than "
+                             f"{self.max_amount}.")
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
```

Comment:
The `SlideLimitationValidator` class is intended to be an abstract base class but currently lacks any abstract methods. Consider adding abstract methods to enforce implementation in subclasses.